src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -326,50 +325,21 @@ interface IOptionSettlementEngine {
     //////////////////////////////////////////////////////////////*/
 
     /**
-     * @notice Returns Option struct details about a given tokenID if that token is
-     * an option.
-     * @param tokenId The id of the option.
-     * @return optionInfo The Option struct for tokenId.
-     */
-    function option(uint256 tokenId) external view returns (Option memory optionInfo);
-
-    /**
-     * @notice Returns information about the exercised and unexercised assets associated with
-     * an options lot claim.
+     * @notice Gets information for a given claim token id.
      * @param claimId The id of the claim
-     * @return claim The Claim struct reflecting information about the options lot.
+     * @return claim The Claim struct for claimId if the tokenId is Type.Claim.
      */
     function claim(uint256 claimId) external view returns (Claim memory claim);
 
     /**
-     * @notice Information about the position underlying a token, useful for determining value.
-     * When supplied an Option Lot Claim id, this function returns the total amounts of underlying
-     * and exercise assets currently associated with a given options lot.
-     * @param tokenId The token id for which to retrieve the Underlying position.
-     * @return underlyingPositions The Underlying struct for the supplied tokenId.
-     */
-    function underlying(uint256 tokenId) external view returns (Underlying memory underlyingPositions);
-
-    /**
-     * @notice Returns the token type (e.g. Option/Claim) for a given token Id
-     * @param tokenId The id of the option or claim.
-     * @return typeOfToken The enum Type of the tokenId.
-     */
-    function tokenType(uint256 tokenId) external view returns (Type typeOfToken);
-
-    /**
-     * @notice Check to see if an option is already initialized.
-     * @param optionKey The option key to check.
-     * @return initialized Whether or not the option is initialized.
+     * @notice Gets option type information for a given tokenId.
+     * @param tokenId The token id for which to get option inf.
+     * @return option The Option struct for tokenId if the optionKey for tokenId is initialized.
      */
-    function isOptionInitialized(uint160 optionKey) external view returns (bool initialized);
-
-    /*//////////////////////////////////////////////////////////////
-    //  Token ID Encoding
-    //////////////////////////////////////////////////////////////*/
+    function option(uint256 tokenId) external view returns (Option memory option);
 
     /**
-     * @notice Encode the supplied option id and claim id
+     * @notice Gets the TokenType for a given tokenId.
```

Comment:
Throughout in the prose, we should standardize on token id / option id / claim id OR tokenId / optionId / claimId, OR something else. Right now it varies throughout, and especially noticeable here bc the stylization differs on this individual function's NatSpec.

Slight preference for tokenId / tokenIds etc., but not strong